### DISCUSSION TITLE
👩‍🌾 Sunflower Fashion Week #5

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#2899] [Art]
Welcome to the 5th Bumpkin Competition a.k.a Sunflower Fashion Week! 💃

The Art Team will choose the best outfits and announce the winners on the first week of every month! 😍

There's no specific theme for this competition! Designers are encouraged to use their imagination and come up with original ideas. 

Before continuing, it is recommended to read the Bumpkin Guideline, which can be found here:

https://docs.sunflower-land.com/contributing/for-designers/bumpkin-guidelines

## Requirements ✏️

- Dimensions of file should be 100x100px.
- The layer (e.g. shirt) should be on it's own layer without any other parts. When placed on top of a Bumpkin it should perfectly align with its body.
- Art should be original and not plagiarised.
- If submitting multiple items, an aseprite file would be appreciated to speed up review. 🤗
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely. Players may trade your item without the need to pay you royalties. The community may make changes and improve artwork that is submitted.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/bumpkin-art-competition#terms-and-conditions) of the game.
- Bumpkins live inside of the game! Please include an NPC version of your design that can be used inside of Sunflower Land.

When submitting art, please include the following:

- Your farm ID.
- A minified version of your art piece.
- An example of the art used inside of the game or on the Bumpkin. Here's a nice submission example from a community artist: https://github.com/sunflower-land/sunflower-land/discussions/2639#discussioncomment-6238691

## Rewards 🏆

The winner will receive 1000 SFL and a runner-up will receive 500 SFL, however this competition is not just about the prize money. It's an opportunity to showcase your incredible talent and become an essential part of the Sunflower Land experience. 

We want to emphasize that even if you don't win, your design may still have the chance to be selected in the future. Our team will carefully review all submissions and may reach out to designers whose work stands out for future collaboration opportunities. Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

Designers have until **September 29th** to submit their entries. After that, our art team will carefully review and analyze each submission to select the winners.

The announcement of the winners will take place during the first week of September.

We can't wait to see what amazing designs you come up with. 

Good luck to all participants! 🌻

**Comment by ITsMeRBO2** [comment]
Hey Hope You guys enjoy The idea 
- Ninja Bumpkin 
![GIF ninja all](https://github.com/sunflower-land/sunflower-land/assets/125754949/ab31a1a8-bf59-49f3-9d48-b70f4de45095)


👉 Black Ninja : 
![layers2](https://github.com/sunflower-land/sunflower-land/assets/125754949/bc68fb20-3c4b-4fa8-8e02-36f080ec6fd4)

👉 Red Ninja : 
![layers3](https://github.com/sunflower-land/sunflower-land/assets/125754949/121daddf-7040-4b9d-a48e-4e7c0918bbd9)

👉 Gray Ninja : 
![layers1](https://github.com/sunflower-land/sunflower-land/assets/125754949/3d46be1f-bdde-4eb8-b7dd-1c44d52a6649)


NPC GIF : 
![gif ninja](https://github.com/sunflower-land/sunflower-land/assets/125754949/73c9c982-8a96-4ffc-9843-caa6f2964c07)


MY ID LAND  :  143993 
My discord    :   itsmerb.

**Comment by ITsMeRBO2** [comment]
Hello There They call me Doctor Farmer what about you 

- Doctor Bumpkin : 
![final de final](https://github.com/sunflower-land/sunflower-land/assets/125754949/dd11c90a-c269-4322-a874-7b2cd0d58619)

👉 Hat : 
![hat 2](https://github.com/sunflower-land/sunflower-land/assets/125754949/d089c431-4b8f-44eb-8111-9d2bb28606d6)

👉 Mask : 
![mask 2](https://github.com/sunflower-land/sunflower-land/assets/125754949/b231726c-9dda-4652-ab0f-1b575eadc627)

👉 Bunddle : 

![bundel 2](https://github.com/sunflower-land/sunflower-land/assets/125754949/3a2a5ce8-a949-4206-b51c-b7648477d5ed)

👉 NPC : 
![gif doc](https://github.com/sunflower-land/sunflower-land/assets/125754949/ba777aed-9ae8-4863-979b-8a7b23c78431)



MY ID LAND : 143993
My discord : itsmerb.

**Comment by andand0** [comment]
### Oktoberfest set

`Farm ID: 165810`
`Discord: Andando`

![Oktoberfest preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/4feae570-c883-4aa7-a789-82e4c216e171)

<br>

| Background| Hat | Shirt | Lederhosen | Shoes | Sausage |
|--------|--------|--------|--------|--------|--------|
|![Oktoberfest background](https://github.com/sunflower-land/sunflower-land/assets/142132586/f5afa17e-7d9e-4c9a-a098-3b4eacf7e875) |![Oktoberfest hat](https://github.com/sunflower-land/sunflower-land/assets/142132586/5dd94a3c-c32f-4cf1-b7ad-9b58109517a6) |![Oktoberfest shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/fcf86432-f379-4cba-a9c8-2b073784f66b)| ![Oktoberfest pants - lederhosen](https://github.com/sunflower-land/sunflower-land/assets/142132586/9dfda18f-cf19-4019-8633-6f777b348ab2) | ![Oktoberfest pants - shoes](https://github.com/sunflower-land/sunflower-land/assets/142132586/6e182b78-de46-423a-aa6b-6b9889354639) | ![Oktoberfest tool - saussage](https://github.com/sunflower-land/sunflower-land/assets/142132586/99ceea4e-8679-4470-bdc3-a1c71db511fd) | 

<br> <hr>

### NPC Version

<br>

![Oktoberfestt-NPC-preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/17ba0c26-d189-43d2-b1b1-d143a1f00bf1)

| GIF | Spritesheet |
|--------|--------|
|![Oktoberfest-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/f6622ba9-38d8-467f-97a4-6675b2e2681e) |![Oktoberfest-NPC-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/2f14dba3-cfd8-469a-a4e0-81aa64bd3eb8) | 

| Sprite Hat | Sprite Shirt | Sprite Pants | Sprite Shoes |
|--------|--------|--------|--------|
|![Oktoberfest-NPC hat-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/6d2f6ac3-676a-42ed-9384-c952e8c0a0bc)|![Oktoberfest-NPC shirt-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/7f2793a8-2b57-4aaf-8954-2aa4ef38715b) | ![Oktoberfest-NPC pants lederhosen-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/199e5ac2-c718-4e18-93cd-f6b111a8a184) | ![Oktoberfest-NPC shoes-spritesheet](https://github.com/sunflower-land/sunflower-land/assets/142132586/ea1d5451-e3a7-4e1f-80c3-82ece6f0f81c) | 


<br><hr><br>

### Astronaut outfit

<br>

![astronaut set](https://github.com/sunflower-land/sunflower-land/assets/142132586/03cce1f0-fa68-426a-b363-e8c77308dd84)

<br>

| Helmet| shirt | Pants | Shoes | Tool |
|--------|--------|--------|--------|--------|
|![astronaut helmet](https://github.com/sunflower-land/sunflower-land/assets/142132586/30763118-3284-49cf-98c8-ab87ae9214a9) | ![astronaut shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/a64e9cd6-8799-43b6-9237-befa0380e9a4) | ![astronaut pants](https://github.com/sunflower-land/sunflower-land/assets/142132586/5c47a024-389b-485e-a09d-9c73ec06e0b3) |![astronaut shoes](https://github.com/sunflower-land/sunflower-land/assets/142132586/e1b25393-ab9c-433d-ac3c-1fb4778c0c2d) | ![astronaut flag](https://github.com/sunflower-land/sunflower-land/assets/142132586/6ed20182-1716-4af9-9a40-be236881ef0c) | 

<br><hr> <br>

### NPC Version

![Astronaut NPC preview](https://github.com/sunflower-land/sunflower-land/assets/142132586/f5b831dc-002b-4e82-9fd8-af5995e8762b)

| Gif| Sprite|
|--------|--------|
|![astronaut-NPC v1](https://github.com/sunflower-land/sunflower-land/assets/142132586/5c6eba95-3952-42ce-9c43-5d38ba3b598f) |![astronaut-NPC](https://github.com/sunflower-land/sunflower-land/assets/142132586/65b4c345-7998-4b05-b604-668bf88dc47e) | 


<br>

| Helmet| Shirt| Pants| Shoes|
|--------|--------|--------|--------|
|![astronaut-NPC hat](https://github.com/sunflower-land/sunflower-land/assets/142132586/e0abfb2a-9ce8-4479-b7aa-825486900626)  | ![astronaut-NPC-shirt](https://github.com/sunflower-land/sunflower-land/assets/142132586/36c092e2-de56-42c0-b5da-334a0c29b1c6)| ![astronaut-NPC-pants](https://github.com/sunflower-land/sunflower-land/assets/142132586/a343945b-627e-4674-847c-297bb84a6d59)| ![astronaut-NPC-shoes](https://github.com/sunflower-land/sunflower-land/assets/142132586/1dfc3920-1c5c-471a-8062-0a17ea575b7a) | 

<br>

`Farm ID: 165810`
`Discord: Andando`

**Comment by ITsMeRBO2** [comment]
LuiGi bumpkin Is here : 

Name :  - LuiGi Bumpkin
![finaaal 500 ](https://github.com/sunflower-land/sunflower-land/assets/125754949/dac16c60-c6e7-459e-8ee3-70804188a45e)



💭 LuiGi Hat : 
![hat luigi](https://github.com/sunflower-land/sunflower-land/assets/125754949/7c21114b-2c67-4714-aee6-71a4b4f89a6b)

💭 Clothes : 
![labsa da](https://github.com/sunflower-land/sunflower-land/assets/125754949/353ecca6-f87c-4b5a-acc0-6ee24caad713)


💭 Background : 
![bach200](https://github.com/sunflower-land/sunflower-land/assets/125754949/9f660d67-a1a0-4e8f-a28b-9ddd831a7686)



💭 NPC GIF :
![final de final de](https://github.com/sunflower-land/sunflower-land/assets/125754949/ca4feeda-97a9-438d-b1bb-0acd732184c2)



MY ID LAND : 143993
My discord : itsmerb.

**Comment by peperol** [comment]
I'm a sweet apple worm
![IMG_0355](https://github.com/sunflower-land/sunflower-land/assets/114354077/95dcdcb9-9b5d-4bd7-8be1-ed378d1480d1)

suit apple worm
![IMG_0354](https://github.com/sunflower-land/sunflower-land/assets/114354077/a7ba8d48-4764-4d72-9683-3de710ea1910)

NPC
![IMG_0352](https://github.com/sunflower-land/sunflower-land/assets/114354077/053442bb-ceaa-4d81-9d6c-bfa1cdc0b89d)

in land
![IMG_0353](https://github.com/sunflower-land/sunflower-land/assets/114354077/945f9bba-df1d-4e87-9a72-6063133bef2d)


farm ID 145849
discord peperol

**Comment by Vergelsxtn** [comment]
Farm ID: 63
Discord ID: Vergelsxtn

Chickee Onesie 🐔💨

Bumpkin Prev:
![chicky-onesie-PREV](https://github.com/sunflower-land/sunflower-land/assets/99605039/e69f3a8c-bf87-4bd5-b5fa-785a0fb46907)
 
NPC Prev:
![island](https://github.com/sunflower-land/sunflower-land/assets/99605039/4575d284-1f96-49b6-b76e-27286b6ac196)

Assets:
| Bumpkin |
|--------|
| ![chicky-onesie](https://github.com/sunflower-land/sunflower-land/assets/99605039/a6b9cc3c-16b3-40dd-b2d6-49f0e688afa2)| 

| NPC|
|--------|
| ![chicky-onesie-NPC-Sheet](https://github.com/sunflower-land/sunflower-land/assets/99605039/576b4791-36d8-487c-8663-db82e808c50a)|
